tests/conftest.py

```diff
@@ -132,7 +136,7 @@ def backend(monkeypatch):
         m.setattr(dummy_backend, "squeeze", lambda r, modes: None)
         m.setattr(dummy_backend, "rotation", lambda r, modes: None)
         m.setattr(dummy_backend, "beamsplitter", lambda t, r, m1, m2: None)
-        m.setattr(dummy_backend, "measure_homodyne", lambda phi, modes, select, shots: 5)
+        m.setattr(dummy_backend, "measure_homodyne", lambda phi, modes, select, shots: np.array([[5]]))
```

Comment:
I know I'm in the minority, but I don't like having to use singleton axes :frowning_face:

Reply:
I agree that it looks a bit ugly, but it might be a better long term solution. If e.g. later wanting to add support for measuring more modes and/or shots it would make more sense since they would all have the same shape. 

Maybe an option could be (as @josh146 suggested) having a sample class that could return things to the user in a "nicer" way, while still returning the samples from the backends with shape `(shots, measured_modes)` and storing them in that way inside the new class?